static/styles/compose.css

```diff
@@ -276,19 +276,20 @@
     position: absolute;
 }
 
-.compose_resolved_topic_close,
```

Comment:
Is there any reason to move this line below? We try to avoid such unnecessary changes to keep the git history clean. (Same for other couple of changes too in this file).

Reply:
done - I changed this to make the order of classnames consistent.